libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/trip/session/MapboxTripSession.kt

```diff
@@ -184,7 +184,6 @@ class MapboxTripSession(
         enhancedLocation = null
         routeProgress = null
         isOffRoute = false
-        navigator.reset()
```

Comment:
is this the fix?

Reply:
Correct so we only reset the `Navigator` when in `onDestroy`